ts/components/session/icon/index.tsx

```diff
@@ -0,0 +1,3 @@
+import { icons, SessionIconSize, SessionIconType } from './Icons';
```

Comment:
You can do the same thing with `SessionIcon` and `SessionIconButton`. This makes it so they're all namespaced under `session/icon`.

So from:
```
import { SessionIcon } from '../session/icon/SessionIcon';
import { SessionIconSize, SessionIconType } from '../session/icon';
```
it would become:
```
import { SessionIcon, SessionIconSize, SessionIconType } from '../session/icon';
```

Reply:
Please see my namespacing now.